lib/msal-common/src/telemetry/server/ServerTelemetryManager.ts

```diff
@@ -28,17 +33,21 @@ export class ServerTelemetryManager {
         this.wrapperVer = telemetryRequest.wrapperVer || Constants.EMPTY_STRING;
 
         this.telemetryCacheKey = SERVER_TELEM_CONSTANTS.CACHE_KEY + Separators.CACHE_KEY_SEPARATOR + telemetryRequest.clientId;
+
+        // Initialize the cache outcome value
+        this.setCacheOutcome(this.forceRefresh ? CacheOutcome.FORCE_REFRESH : CacheOutcome.NO_CACHE_HIT);
```

Comment:
This should be done wherever the cache lookup is done instead (e.g. SilentFlowClient)